<launch>
  <!-- <include file="$(find scout_gazebo_sim)/launch/scout_smaze_world.launch">
  </include> -->

  <include file="$(find aloam_velodyne)/launch/aloam_velodyne_HDL_32.launch">
	  <arg name="rviz" value="false"/>
  </include>

  <node pkg="tf" type="static_transform_publisher" name="world_to_camera_init" args="0 0 0 0 0 0 world camera_init 10" />
  <!-- <node pkg="tf" type="static_transform_publisher" name="camera_to_link" args="0 0 0 0 0 0 aft_mapped base_link 10" /> -->
  <node pkg="tf" type="static_transform_publisher" name="camera_to_link" args="-0.17193 0 -0.13942 0 0 0 aft_mapped base_link 10" />

  <node pkg="waypoint_generator" name="waypoint_generator" type="waypoint_generator">
      <remap from="~goal" to="/goal"/>
      <param name="waypoint_type" value="manual-lonely-waypoint"/>    
  </node>

  <node pkg="planner" type="global_planning_node" name="global_planning_node" output="screen" required = "true">
      <remap from="~waypoints"       to="/waypoint_generator/waypoints"/>
      <!-- <remap from="~map"             to="/cloud_registered"/>  -->

      <rosparam command="load" file="$(find planner)/config/simulation.yaml"/>
  </node>

  <!-- <node pkg="gpr" type="gpr_path" name="gpr_path" output="screen" required="true">
      <param name="file/cfg_path" type="string" value="$(find gpr)/config/hyperparam.txt"/>
  </node> -->
    <!-- <node name="local_planner" pkg="local_planner" type="local_planner.py" launch-prefix="gnome-terminal -x">
  </node> -->

  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-d $(find planner)/config/for_simulation/simulation.rviz" required="true"/> -->
</launch>
